File: Involucrate/Models/ONG.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Involucrate.Models
{
    public class ONG
    {
        public int IdOng { get; set; }
        public int IdUsuario { get; set; }
        public string NombreOng { get; set; }
        public string Descripcion { get; set; }
        public string Direccion { get; set; }
        public string Telefono { get; set; }
        public string CorreoElectronico { get; set; }
        public string Url { get; set; }

    }
}

File: app/src/main/java/com/andrea_murillo/involucrate/model/Ong.kt
package com.andrea_murillo.involucrate.model

data class Ong(
    val id: Long? = null,
    val usuario: Usuario,
    val nombreOng: String,
    val descripcion: String,
    val direccion: String,
    val telefono: String,
    val imagen: String? = null,
    val correoElectronico: String,
    val url: String? = null,
    val verificada: Boolean = false
)